src/main/deploy/pathplanner/autos/left-0300.auto

```diff
@@ -17,7 +17,7 @@
               {
                 "type": "named",
                 "data": {
-                  "name": "PrepareScore"
+                  "name": "PrepareL2"
```

Comment:
In right-0300.auto, line 67 has "PrepareL2" but on line 67 of this file it has "PrepareScore". Which is correct?

I see the same differences on line 117 for these files